src/components/Nav/Nav.js

```diff
@@ -1,8 +1,52 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
+import { Link } from 'react-router-dom';
 import './Nav.scss';
 
+import nikeLogo from '../../assets/Nav/nike.png';
+
 const Nav = () => {
-  return <div className="nav">nav</div>;
+  const [navData, setNavData] = useState([]);
+
+  useEffect(() => {
+    fetch('/data/navData.json', {
+      method: 'GET',
+    })
+      .then(res => res.json())
+      .then(data => {
+        setNavData(data);
+      });
```

Comment:
```suggestion
      .then(setNavData);
```
then의 callback에서 처리할 코드가 없고, 받은 데이터를 바로 setState 해주는 경우라면 이렇게도 쓸 수 있습니다.
주의할건, 모든 상황에 적용되는게 아닌, 단순 setState만 필요한 경우에만 이렇게 사용할 수 있습니다.

Reply:
이렇게도 쓰이는군요 👀 감사합니다!
피드백 주신 부분 수정 완료 후 현재 PR에 commit으로 추가했습니다 :)